rfc/text/advanced/ap_10_rpc_progressive_call_invocations.md

```diff
@@ -247,7 +247,7 @@ When a *Callee* receives an INVOCATION under the following conditions:
 - the INVOCATION request ID does not correspond to a new call request, and,
 - the INVOCATION request ID does not match any RPC invocation in progress,
 
-then it MUST ignore and discard that INVOCATION message without any further correlated response to the *Dealer*. The rationale for this is that the *Caller* will eventually receive a correlated RESULT or ERROR message from the previous call completion and will be able to handle the call completion accordingly.
+then it MUST ignore and discard that INVOCATION message without any further correlated response to the *Dealer*. The rationale for this is that the *Dealer* will eventually receive a correlated YIELD or ERROR message from the *Callee* and then send a correlated RESULT or ERROR message to the *Caller*, thus the *Caller* will be able to handle the call completion accordingly.
```

Comment:
Although this is ok enough, to be very pedantic, I still think this dealer-callee race is a potential problem for the dealer more so than for the caller...
Maybe as a compromise:

> ... thus the *caller* and the *dealer* will be able to ...

?
But again, this is ok enough for me as well, since it's not a critical piece of the docs.

Reply:
Applied suggested change.